### PR TITLE
Prepare for a 0.1.0 release.

### PR DESCRIPTION
**Pull request by ltratt**
As always there is much more that we could do before a release, but this seems better than nothing. The commits are hopefully self-contained.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# grmtools 0.1.0 (2019-05-16)
```
Comment (jacob-hughes) [inline CHANGES.md]: Copy/paste leftover?

Comment (ltratt) [reply CHANGES.md]: Oops!

**Comment by ltratt** [comment]
That new commit should fix things.

**Comment by jacob-hughes** [comment]
Cool, LGTM. Please squash!

**Comment by ltratt** [comment]
Squashed!

**Pull request merged**